src/index.js

```diff
@@ -7,8 +7,10 @@ import './styles/index.css'
 import { unregister } from './registerServiceWorker'
 import { RehydrationProvider } from './components'
 import ReactGA from 'react-ga'
-ReactGA.initialize(process.env['REACT_APP_GA_TRACKINGID'])
-ReactGA.pageview(window.location.pathname + window.location.search)
+if (process.env['REACT_APP_GA_TRACKINGID']) {
```

Comment:
wouldn't it be better to check if we're running in `production`?
the `NODE_ENV` should be available:
https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables
>  There is also a special built-in environment variable called `NODE_ENV`. You can read it from `process.env.NODE_ENV`. When you run `npm start`, it is always equal to `'development'`, when you run `npm test` it is always equal to `'test'`, and when you run `npm run build` to make a production bundle, it is always equal to `'production'`.

Reply:
Sure, much better, thanks for the hint